.kuzzlerc.sample

```diff
@@ -177,6 +177,11 @@
   // Configuration for Kuzzle server
   /*
   "server": {
+    // * http disabled: Defaults to false. Set true to disable http support in Kuzzle
```

Comment:
HTTP support will be removed from Kuzzle when the following PR is merged: https://github.com/kuzzleio/kuzzle/pull/541